localnet/poktrolld/config/relayminer_config.yaml

```diff
@@ -16,7 +16,7 @@ suppliers:
     proxy_names:
       - http-proxy
     hosts:
-      - tcp://relayminers:8545
+      - http://relayminers:8545
```

Comment:
I'm still seeing a lot of tcp://relayminer through.

I don't have full context but wanted to check if those should be left as is?

![Screenshot 2024-03-14 at 12 33 45 PM](https://github.com/pokt-network/poktroll/assets/1892194/a9174d14-52cc-44b2-9255-def43551042a)

Reply:
Good question. These values just need to match the service configuration endpoint that the suppliers are staked for. I'll take a look today.